pkgs/desktops/plasma-5/aura-browser.nix

```
@@ -0,0 +1,18 @@
+{
+  mkDerivation,
+  extra-cmake-modules,
+  qtwebengine,
+  qtquickcontrols2,
+  kirigami2,
+  ki18n,
+}:
```

```suggestion
{
, mkDerivation
, extra-cmake-modules
, qtwebengine
, qtquickcontrols2
, kirigami2
, ki18n
}:

```

Yes, I know the other kde packages are like this, too but we should format them to follow the vast majority of nixpkgs.